Gems/GenAIAmazonBedrockVendorBundle/Code/Source/Providers/Claude/ClaudeAmazonBedrockProvider.h

```diff
@@ -7,11 +7,13 @@
  */
 
 #pragma once
+#include <GenAIAmazonBedrockVendorBundle/GenAIAmazonBedrockVendorBundleTypeIds.h>
```

Comment:
```suggestion

#include <GenAIAmazonBedrockVendorBundle/GenAIAmazonBedrockVendorBundleTypeIds.h>
```

Nit: newline after #pragma once